android/src/main/java/com/tailscale/ipn/ui/viewModel/MainViewModel.kt

```diff
@@ -81,7 +94,7 @@ class MainViewModel : IpnViewModel() {
     viewModelScope.launch {
       var previousState: State? = null
 
-      combine(Notifier.state, vpnPrepared) { state, prepared -> state to prepared }
+      combine(Notifier.state, isVpnPrepared) { state, prepared -> state to prepared }
```

Comment:
Can we just reference `vpnViewModel.vpnPrepared` instead of adding an extra new `val` which only gets used once?

Reply:
mainViewModel.isVpnPrepared is also used by MainView